package/kedro_viz/launchers/cli/run.py

```diff
@@ -164,17 +165,19 @@ def run(
             "is_lite": lite,
         }
         if autoreload:
-            from watchgod import RegExpWatcher, run_process
+            from watchfiles import run_process
 
+            run_process_args = [str(kedro_project_path)]
```

Comment:
any reason why path is removed from kwargs and made as args ?

Reply:
Because in new `watchfiles`, `run_process` does not support path as keyword argument.